### DISCUSSION TITLE
failed get objs: failed to list objs: permissiondenied:permission not exists

### DISCUSSION THREAD

**Discussion by Setsuna924** [AlistGo/alist#4803] [Q&A]
挂载阿里云盘open，工作状态正常（work），但在主页里点进去显示failed get objs: failed to list objs: permissiondenied:permission not exists，不知如何解决。另有案例在SMB连接下更改根目录名称解决，但我这里仍未解决

**Comment by li-peifeng** [comment]
先取消所有授权，再重新申请刷新令牌，反正我就是这么搞好的

**Comment by muxinxy** [comment]
解决了，删除存储重新添加

**Reply by BlueSkyXN** [reply]
> 解决了，删除存储重新添加


研究半天原来是这个吊错误，删除后重新加解决
